[EH] Changelog (#37586)
Azure/azure-sdk-for-python: sdk/eventhub/azure-eventhub/CHANGELOG.md

# Release History

## 5.12.2 (2024-10-02)

### Bugs Fixed

> - Fixed a bug where async consumer memory usage had no backpressure implemented, causing a memory leak. ([#36398](https://github.com/Azure/azure-sdk-for-python/issues/36398))

Reviewer: ```suggestion
- Implemented backpressure for  async consumer  to address a memory leak issue. ([#36398](https://github.com/Azure/azure-sdk-for-python/issues/36398))
```
